tests/suites/test_suite_x509parse.function

```diff
@@ -1103,7 +1114,7 @@ exit:
 }
 /* END_CASE */
 
-/* BEGIN_CASE depends_on:MBEDTLS_X509_USE_C */
+/* BEGIN_CASE depends_on:MBEDTLS_X509_USE_C:MBEDTLS_X509_INFO */
 void x509_oid_desc( data_t * buf, char * ref_desc )
 {
     mbedtls_x509_buf oid;
```

Comment:
Because the original author (@pkolbus) signed a CLA when this commit was first made, a DCO from the original author is not needed